editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/SubmissionsTable.tsx

```diff
@@ -0,0 +1,138 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import React from "react";
+
+import { SubmissionData } from "./submissionData";
+
+const StyledTableContainer = styled(TableContainer)(({ theme }) => ({
+  border: `1px solid ${theme.palette.divider}`,
+  overflow: "auto",
+  "&:focus": {
+    outline: `2px solid ${theme.palette.primary.main}`,
+  },
+}));
+
+const StyledTable = styled(Table)({
+  borderCollapse: "collapse",
+});
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  border: `1px solid ${theme.palette.divider}`,
+}));
+
+const DividerStyled = styled(Box)({
+  paddingBottom: "8px",
+  marginBottom: "8px",
+  borderBottom: "1px solid #eee",
+  "&:last-child": {
+    border: "none",
+    paddingBottom: "0",
+    marginBottom: "0",
+  },
+});
+
+function formatDate(date: Date | string) {
+  if (date === "N/A") return date;
+  return new Date(date).toLocaleDateString("en-UK");
+}
+
+interface SubmissionsTableProps {
+  applications: SubmissionData[];
+}
+
+const SubmissionsTable: React.FC<SubmissionsTableProps> = ({
+  applications,
+}) => {
+  return (
+    <StyledTableContainer>
+      <StyledTable stickyHeader aria-label="sticky table">
+        <TableHead>
+          <TableRow>
+            {[
+              "Session ID",
+              "Submitted At",
+              "Payment Requests",
+              "Payment Status",
+              "BOPS Applications",
+              "Uniform Applications",
+              "Email Applications",
+            ].map((headCell) => (
+              <StyledTableCell key={headCell}>{headCell}</StyledTableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {applications.map((row) => (
+            <TableRow key={row.sessionId}>
+              <StyledTableCell component="th" scope="row">
+                {row.sessionId}
+              </StyledTableCell>
+              <StyledTableCell>{formatDate(row.submittedAt)}</StyledTableCell>
+              <StyledTableCell>
+                {row.paymentRequests && row.paymentRequests.length > 0
+                  ? row.paymentRequests.map((request, index) => (
+                      <DividerStyled key={index}>
+                        ID: {request.id}, Created At:{" "}
+                        {formatDate(request.createdAt)}, Paid At:{" "}
+                        {formatDate(request.paidAt)}, Gov Payment ID:{" "}
+                        {request.govPaymentId}
+                      </DividerStyled>
+                    ))
+                  : "None"}
+              </StyledTableCell>
```

Comment:
I think when we're more confident on how we want to render multiple values in one cell I think we could maybe abstract this into a function which takes in an array of values, maps to formatting and renders the results? 

Although maybe want to have a better idea of the designs first?

Reply:
Come to think of it I think I can do basic tests for ids to at least show some of the data is rendered

Reply:
Fixed: 4ee7a5ee8e7bd762b7c93e5db383b25edd241bf9